Tasks/XamariniOSV2/Tests/L0.ts

```diff
@@ -14,6 +14,24 @@ describe('XamariniOS L0 Suite', function () {
 
     });
 
+    it('run XamariniOSV2 with all default inputs', (done) => {
+        this.timeout(1000);
+
+        const tp = path.join(__dirname, 'L0DefaultInputs.js');
+        const tr = new ttm.MockTestRunner(tp);
+
+        tr.run();
+
+        assert(tr.ran('/home/bin/nuget restore src/project.sln'), 'it should have run nuget restore');
+        assert(tr.ran('/home/bin/msbuild src/project.sln /p:Configuration=Release /p:Platform=iPhone'), 'it should have run msbuild');
+        assert(tr.invokedToolCount == 3, 'should have only run 3 commands');
+        assert(tr.stderr.length == 0, 'should not have written to stderr');
+        assert(tr.succeeded, 'task should have succeeded');
+
+        done();
```

Comment:
You don't need to call `done` unless you are running async code.

Reply:
Inside the function, done() needs to be called.